x-pack/plugins/observability_solution/inventory/kibana.jsonc

```diff
@@ -16,7 +16,8 @@
       "features",
       "unifiedSearch",
       "data",
-      "share"
+      "share",
+      "spaces"
```

Comment:
`spaces` should be an `optionalPlugin`. See [here](https://github.com/elastic/kibana/issues/149687)

Reply:
I wasn’t aware of this, thanks you! 
I’ll make the necessary adjustments.

Reply:
@crespocarlos the changes have been included in this [commit](https://github.com/elastic/kibana/pull/195557/commits/a2239e00c0e1ad493bbf52c2948f5d471c4038f6).